static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -0,0 +1,191 @@
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import Card from 'sentry/components/card';
+import ExternalLink from 'sentry/components/links/externalLink';
+import {Tooltip} from 'sentry/components/tooltip';
+import {IconCommit, IconGithub, IconInfo, IconMail} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {MissingMember} from 'sentry/types';
+
+type Props = {
+  missingMembers: {integration: string; users: MissingMember[]};
+  onSendInvite: (email: string) => void;
+};
+
+export function InviteBanner({missingMembers, onSendInvite}: Props) {
+  // NOTE: this is currently used for Github only
+  // TODO(cathy): include snoozing, docs link
+
+  if (!missingMembers?.users) {
+    return null;
+  }
+
+  const cards = missingMembers?.users.slice(0, 5).map(member => (
+    <MemberCard key={member.userId} data-test-id={`member-card-${member.userId}`}>
+      <MemberCardContent>
+        <MemberCardContentRow>
+          <IconGithub size="sm" />
+          <StyledExternalLink href={`http://github.com/${member.userId}`}>
+            {tct('@[userId]', {userId: member.userId})}
+          </StyledExternalLink>
+        </MemberCardContentRow>
+        <MemberCardContentRow>
+          <IconCommit size="xs" />
+          {tct('[commitCount] Recent Commits', {commitCount: member.commitCount})}
+        </MemberCardContentRow>
+        <Subtitle>{member.email}</Subtitle>
+      </MemberCardContent>
+      <Button
+        size="sm"
+        onClick={() => onSendInvite(member.email)}
+        data-test-id="invite-missing-member"
+        icon={<IconMail />}
+      >
+        {t('Invite')}
+      </Button>
+    </MemberCard>
+  ));
+
+  cards?.push(
+    <MemberCard key="see-more" data-test-id="see-more-card">
+      <MemberCardContent>
+        <MemberCardContentRow>
+          <SeeMoreContainer>
+            {tct('See all [missingMembersCount] missing members', {
+              missingMembersCount: missingMembers?.users.length,
+            })}
+          </SeeMoreContainer>
+        </MemberCardContentRow>
+        <Subtitle>
+          {tct('Accounting for [totalCommits] recent commits', {
+            totalCommits: missingMembers?.users.reduce(
+              (acc, curr) => acc + curr.commitCount,
+              0
+            ),
+          })}
+        </Subtitle>
+      </MemberCardContent>
+      <Button
+        size="sm"
+        priority="primary"
+        // TODO(cathy): open up invite modal
+        data-test-id="view-all-missing-members"
+      >
+        {t('View All')}
+      </Button>
+    </MemberCard>
+  );
+
+  return (
+    <StyledCard data-test-id="invite-banner">
```

Comment:
Can everyone invite? If invite needs permission we probably shouldn't show the card to those who cannot invite.

Reply:
Just saw below that permission is in parent component. It's better to be in the component itself so someone who doesn't have this knowledge doesn't reuse your component by mistake. Also if permission changes then you have only one place to update it.